kalite/distributed/templates/distributed/base.html

```diff
@@ -54,7 +54,7 @@
         {% endif %}
 
         <!-- JS translation files dynamically generated by django, then saved to the static file below -->
-        <script type="text/javascript" src="{% static 'js/i18n/'|add:request.language|add:'.js' %}"></script>
+        <script type="text/javascript" src="{% static 'js/i18n/'|add:request.language|add:'.js' True %}"></script>
```

Comment:
We now let the browser cache the JS translation catalog. This is because this file is really big now: 18.7 MB here in my system.